src/lib/viewers/archive/constants.js

```diff
@@ -9,4 +9,6 @@ const VIEWS = {
     VIEW_SEARCH: 'search',
 };
 
-export { TABLE_COLUMNS, VIEWS };
+const ROOT_FOLDER = 'root/';
```

Comment:
Can this be defined at the top of the file? Is this folder injected into the manifest by conversion? If so, can you add a comment stating such?

Reply:
> Can this be defined at the top of the file? Is this folder injected into the manifest by conversion? If so, can you add a comment stating such?

This folder is not injected by conversion. So it can be named 'root/' or 'rootFolder/' or whatever we want.